app/controllers/mixins/ems_common_angular.rb

```diff
@@ -360,6 +369,12 @@ def build_credentials(ems)
         amqp_password = params[:amqp_password] ? params[:amqp_password] : ems.authentication_password(:amqp)
         creds[:amqp] = {:userid => params[:amqp_userid], :password => amqp_password}
       end
+      # TODO params[:ceilometer_userid] will be nil until we add ceilometer authentication in the UI
+      # TODO if ems.supports_authentication?(:ceilometer) && params[:ceilometer_userid]
+      if params[:event_stream_selection] == "ceilometer"
+        # TODO ceilometer_password = params[:ceilometer_password] ? params[:ceilometer_password] : ems.authentication_password(:ceilometer)
+        creds[:ceilometer] = {:userid => "", :password => ""}
+      end
```

Comment:
the auth for Ceilometer will be always inherited from default, so this should not be needed